Marlin/src/lcd/e3v2/common/encoder.cpp

```diff
@@ -169,13 +169,15 @@ EncoderState encoderReceiveAnalyze() {
 #if PIN_EXISTS(LCD_LED)
 
   // Take the low 24 valid bits  24Bit: G7 G6 G5 G4 G3 G2 G1 G0 R7 R6 R5 R4 R3 R2 R1 R0 B7 B6 B5 B4 B3 B2 B1 B0
-  uint16_t LED_DataArray[LED_NUM];
+  uint32_t LED_DataArray[LED_NUM];
```

Comment:
Did this code work at all before, or were you prevented from even trying it due to the errors?

I am wondering what the old behavior was here, with a 16-bit integer trying to hold a 24-bit value?